openassessment/xblock/openassessmentblock.py

```diff
@@ -447,6 +456,10 @@ def _create_ui_models(self):
         """
         ui_models = [UI_MODELS["submission"]]
         for assessment in self.valid_assessments:
+            if assessment["name"] == "staff-assessment" and assessment["required"] == False:
+                # Check if staff have graded the assessment
```

Comment:
I think this is. It looks like we probably want to render it if the staff assessment exists.